### PR TITLE
Updated docs + PyPI compatibility.

### PR DESCRIPTION
**Pull request by pv101**

### CODE REVIEW

File: doc/src/home.rst [hunk 1]
```diff
@@ -16,23 +16,16 @@ of hardware platforms via use of an in-built domain specific language
...
-- Solution files produced - Unstructured VTK (.vtu)
-
-What is PyFR Not?
------------------
-
-PyFR is not a fully fledged 'production' flow solver with all the
-associated bells and whistles. Additionally, while we will do our best
-to help new users, no level of support is guaranteed!
+- Mesh Files Imported - Gmsh (.msh)
+- Solution Files Exported - Unstructured VTK (.vtu)
```
Comment (FreddieWitherden) [inline doc/src/home.rst]: .ptvu

File: setup.py [hunk 2]
```diff
@@ -109,23 +109,29 @@
...
+developed in the Vincent Lab, Department of Aeronautics, Imperial
+College London, UK.'''
 
 setup(name='pyfr',
       version=version,
       description='Flux Reconstruction in Python',
       long_description=long_description,
       author='Imperial College London',
+      author_email='info@pyfr.org',
+      url='www.pyfr.org',
```
Comment (FreddieWitherden) [inline setup.py]: Do we need an http:// prefix here?

Comment (pv101) [inline setup.py]: It didn't seem to care, but I can add it.

File: doc/src/home.rst [hunk 3]
```diff
@@ -16,23 +16,16 @@ of hardware platforms via use of an in-built domain specific language
...
-- Spatial discretisation - High-order flux reconstruction
-- Temporal discretisation - Explicit Runge-Kutta
+- Platforms - CPU Clusters, Nvidia GPU Clusters, AMD GPU Clusters
+- Spatial Discretisation - High-Order Flux Reconstruction
+- Temporal Discretisation - Explicit Runge-Kutta
 - Precision - Single, Double
-- Mesh files read - Gmsh (.msh)
-- Solution files produced - Unstructured VTK (.vtu)
-
-What is PyFR Not?
```
Comment (FreddieWitherden) [inline doc/src/home.rst]: Good riddance!

**Pull request closed without merging**